pkgs/data/fonts/noto-fonts/default.nix

```diff
@@ -125,4 +125,27 @@ in
       maintainers = with maintainers; [ mathnerd314 ];
     };
   };
+  noto-fonts-emoji-blob = stdenv.mkDerivation rec {
```

Comment:
Not sure, but shouldn't this be called `something-bin` because it's not a source build?

Reply:
Yes, you're right - I forgot about that when I made this PR but I'll fix it right away!